source/includes/_install_js_intro.md

```diff
@@ -101,6 +102,7 @@ properties | null | The custom properties of the end_user you are attempting to
 twitter_account | null | A Twitter link you can use to tag your account anytime a promoter leaves a comment.
 wootric_close_position | left | The position of the close button of your survey.
 wootric_recommend_target | depends on the language | Your audience.
+ask_permission_to_share_feedback | false | Ask your end users if they wish to be contacted after leaving written feedback.

```

Comment:
This description seems more like a use case from being able to share. I would think the description would be something like, Ask your end users for permission to share their feedback publicly.

Reply:
Makes sense. Thanks Ray!